packages/e2e-tests/scripts/save-examples.ts

```diff
@@ -446,8 +463,7 @@ async function waitUntilMessage(
 
     process.exit(0);
   } catch (error) {
-    console.error("Unexpected error: ", error);
-
+    console.error("☠☠☠ save-examples failed - ", error?.stack || error);
```

Comment:
Why this change?

Logging `error` will print both the error message _and_stack which seems nicer?

Reply:
* Hum... I'm always doing `error.stack`, because for the longest time, I guess I had middleware that did not print errors correctly. Not sure if old Node might also have had problems with it, all I know is that it was a real problem.
* Also note that `Error.stack` includes `message`, so this gets us the same thing.

But I cannot find any evidence of this being a problem here, so I can revert it.